client/coral-configure/components/ConfigureCommentStream.css

```diff
@@ -0,0 +1,37 @@
+.container {
```

Comment:
Earlier we made a decision to handle all comment stream css in default.css for ease of reconfigurability. I'm open to revisiting that decision (it probably generally makes sense to reexamine our strategy for customizing style), but we should be intentional about it before changing norms.